src/common/components/InputField/InputField.tsx

```diff
@@ -0,0 +1,66 @@
+import { ChangeEvent } from "react";
+import classNames from "classnames/bind";
+import styles from "./InputField.module.scss";
+import { PrefixIcon, SuffixIcon, ErrorMessage, Label } from "./index";
+import Input from "./Input/Input";
+
+const cn = classNames.bind(styles);
+
+interface InputFieldProps {
+  name: string;
+  type?: "text" | "email" | "password" | "dropdown" | "search";
+  placeholder?: string;
+  size?: "sm" | "md" | "full";
+  color?: "white" | "gray";
+  isError?: true | false;
+  label?: string;
+  disabled?: true | false;
+  unit?: string;
+  prefix?: string;
+  value?: string;
+  onChange?: (e: ChangeEvent) => void;
+  border?: "solid" | "none";
+  errorMessage?: string;
+}
+
+export default function InputField({
+  name,
+  type = "text",
+  placeholder = "",
+  size = "md",
+  color = "white",
+  isError = false,
+  label,
+  disabled = false,
+  unit,
+  prefix,
+  value,
+  onChange,
+  border = "solid",
+  errorMessage,
+  ...rest
+}: InputFieldProps) {
+  const className: string = cn("default", size, color, isError && "error", border);
```

Comment:
여기서도 타입 자동 추론을 활용하면 좋을 것 같네요~
```suggestion
  const className = cn("default", size, color, isError && "error", border);
```